### ISSUE TITLE
[New Rule] Underscores in Numeric Literals

### ISSUE THREAD

**Issue by zephvr** [MartinThoma/flake8-simplify#150]
## Explanation

Using underscore on large number make for better comprehension.

This feature was introduced with python 3.6 and [PEP515](https://peps.python.org/pep-0515/) describe it.

## Example

```python
# Bad
amount = 10000000.0
addr = 0xCAFEF00D
flags = 0b0011111101001110
flags = int('0b11110000', 2)

# Good
amount = 10_000_000.0
addr = 0xCAFE_F00D
flags = 0b_0011_1111_0100_1110
flags = int('0b_1111_0000', 2)
```

**Comment by Marenz** [comment]
Related: https://github.com/frequenz-floss/flake8-numbers/pull/4/files
